src/configs/index.js

```diff
@@ -107,16 +107,47 @@ export const EMPTY_FIELD = [
   }
 ];
 
-export const LOGIN_USER_ERROR = [
-  {
-    lang: 'uk',
-    value: 'Неправильна електронна адреса або пароль'
-  },
-  {
-    lang: 'eng',
-    value: 'Wrong e-mail address or password'
-  }
-];
+export const LOGIN_USER_ERROR = {
+  WRONG_CREDENTIALS: [
+    {
+      lang: 'uk',
+      value: 'Неправильна електронна адреса або пароль'
+    },
+    {
+      lang: 'eng',
+      value: 'Wrong e-mail address or password'
+    }
+  ],
+  INPUT_NOT_VALID: [
+    {
+      lang: 'uk',
+      value: 'Ви ввели невірні дані'
+    },
+    {
+      lang: 'eng',
+      value: 'You have typed wrong data'
+    }
+  ]
```

Comment:
Move all translations to translations folder

Reply:
moved